InnerEye/ML/reports/classification_report.py

```diff
@@ -49,19 +52,32 @@ class ReportedMetrics(Enum):
     FalseNegativeRate = "false_negative_rate"
 
 
-def get_results(csv: Path) -> LabelsAndPredictions:
+def read_csv_and_filter_prediction_target(csv: Path, prediction_target: str) -> pd.DataFrame:
     """
-    Given a CSV file, reads the subject IDs, ground truth labels and model outputs for each subject.
-    NOTE: This CSV file should have results from a single epoch, as in the metrics files written during inference, not
-    like the ones written while training.
+    Given one of the csv files written during inference time, read it and select only those rows which belong to the
+    given prediction_target. Also check that there is only a single entry per prediction_target per subject in the file.
+    The csv must have at least the following columns (defined in the LoggingColumns enum):
+    LoggingColumns.Hue, LoggingColumns.Patient, LoggingColumns.Label, LoggingColumns.ModelOutput.
```

Comment:
why is label and modeloutput mandatory, if the code does not use them?

Reply:
Moved this comment to the appropriate function docstrings.